designs/administrator/future-openshift/favoriting/index.md

```diff
@@ -3,29 +3,62 @@ parent: Administrator
 
-### Project selector
-![3](img/dropdown.png)
-* Project favorites will be reflected in the project dropdown, but cannot be edited there.
-* Dropdown sections and a horizontal separator will be used to visually distinguish favorite projects and all other projects.
+![project-selector](img/ProjectSelector.png)
 
+When favoriting a project, the user can hover over a star and select it, which will duplicate the menu item so that the favorited item is at the top of the menu, in the “favorites” section, as well as within the “projects” section.
 
-## Roles list
-![4](img/roles1.png)
-* The roles list will load with several roles automatically favorited. These roles are commonly used and pre-favorited to bring them to the attention of the user.
-* Users may set additional favorite roles by clicking the star toggle at the beginning of a row.
-* The role will not change positions in the list when the star is toggled on or off.
+![favoriting-interaction](img/FavoritingInteraction.png)
 
-![5](img/roles2.png)
-* Users may view only their favorite roles by checking the "Show only favorites" checkbox next to the textbox filter.
-* When the checkbox is selected, the toggle filters will be disabled to avoid conflicting filtering behavior.
-* When the checkbox is deselected, the toggle filters should return to their previous states before the checkbox was selected.
+ Interacting with the switch will only affect the projects listed within the “project” section.
+
+![switch-interaction](img/SwitchInteraction.png)
+
+The search bar can be used to search for a project. Assuming the switch is off, hiding system projects, if the inputted string does not match any projects, then we will show this empty state that suggests showing system projects.
+
+![empty-state](img/ProjectSelectorEmptyState.png)
+
+### Project selector edge cases
+
+Assuming the switch is hiding system projects, if a user were to navigate to a resource page that exists in a system project, the project selector will still reflect the project name in the toggle but the project will not be listed within the menu.
+
+![edge-case](img/EdgeCase1.png)
+
+For example, if the user were to select the first pod listed in the openshift-marketplace namespace, they would be brought to the pod’s page where the project selector would reflect the project, even though it is a system project.
+
+![edge-case2](img/EdgeCase2.png)
+
+## Favoriting and hiding system projects in projects and namespaces
+
+On the projects list view and namespaces list view pages, we will include a favorites column to be able to sort by favorites and customize favorites. We will also pre-filter the view so that it is only showing projects requested by “me” and “users”, which ultimately hides system projects.
```

Comment:
If the user opts into seeing system projects in the project selector and they're in that mental mode of "I want to see everything," I wonder if they'd expect to see system projects on this page by default as well. If so, maybe when this page first loads we could check the setting and change the default filter accordingly? Sorry if this aspect was already discussed!

Reply:
This was something we discussed that would possibly be included in user preferences - be able to update that setting everywhere basically